bootstrap-salt.sh

```diff
@@ -2975,6 +2969,11 @@ install_debian_deps() {
     # YAML module is used for generating custom master/minion configs
     __PACKAGES="${__PACKAGES} python-yaml"
 
+    # Debian 9 needs the dirmgr package in order to import the GPG key later
+    if [ "$DISTRO_MAJOR_VERSION" -ge 9 ]; then
+        __PACKAGES="${__PACKAGES} dirmngr"
```

Comment:
Yeah when you install with bootstrap, it fails because that package is missing. The package gets included when installing the packages from repo.saltstack.com just fine, but not in bootstrap. It's a dependency of `debconf-utils`. I didn't think to check the `gnupg1-curl` dependency though.